### ISSUE TITLE
Compile error with fullTemplateTypeCheck: true

### ISSUE THREAD

**Issue by achimha** [madoar/ng2-archwizard#84]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => search github for a similar issue or PR before submitting
```

**Current behavior**
Angular 5.2 introduces a new AoT compiler option to perform a more thorough type check on templates. It is expected that this option will default to true in a future version:

https://blog.angular.io/angular-5-2-now-available-312d1099bd81

Building a project with ng2-archwizard shows one type error:

`node_modules/ng2-archwizard/dist/components/wizard-navigation-bar.component.d.ts.WizardNavigationBarComponent.html(17,11): : Directive GoToStepDirective, Expected 0 arguments, but got 1.`

* **ng2-archwizard version:** 2.1.0

* **Angular version:** 5.2.0

* **Browser:** [all ]

* **Language:** [TypeScript 2.5.2]

**Comment by madoar** [comment]
Thank you for testing the library with angular 5.2!

Taking a quick look at the source code I'm not sure where the error is located.
The message points to both the html template of `WizardNavigationBarComponent` and to `GoToStepDirective`, but I'm not sure where exactly typescript/angular expects 0 arguments but gets 1.

Sadly I have no test project using angular 5.2, so it's hard for me to test this.
Can you provide a short test project using angular 5.2 with which I can experiment to find and remove the error?

**Comment by achimha** [comment]
The same problem might exist in other directives, sorry for the incomplete patch.

**Comment by madoar** [comment]
@achimha no problem. Thanks for your effort!
Today I've pushed `ng2-archwizard` to angular 5 and prepared the project for a new development iteration (i.e. version 3.0.0).
I hope you can merge/rebase the current master branch in your PRs to ensure that the tests run correctly and everything is compatible.
Afterwards I would love to merge them :)

**Comment by achimha** [comment]
Created a new PR https://github.com/madoar/ng2-archwizard/pull/89

**Comment by madoar** [comment]
This was fixed in #89.
